torch/csrc/jit/export.cpp

```diff
@@ -112,15 +113,59 @@ void addAttribute(onnx::NodeProto * n_p, jit::Node * n, jit::Symbol name) {
   }
 }
 
+void encodeType(onnx::TypeProto* t, Node* n) {
```

Comment:
Wouldn't it be better for this to take a TensorType argument, and then move the logic for actually getting it out (`expect`) to the call site (of which there is only one)?

Reply:
Good point. Moved.